client/my-sites/picker/picker.jsx

```diff
@@ -63,13 +74,17 @@ const SitePicker = React.createClass( {
 		window.scrollTo( 0, 0 );
 	},
 
-	handleClickOutside: function() {
+	closePicker: function() {
 		if ( this.props.layoutFocus && this.props.layoutFocus.getCurrent() === 'sites' ) {
-			this.props.layoutFocus && this.props.layoutFocus.set( 'sidebar' );
+			this.props.layoutFocus.set( 'sidebar' );
 			this.scrollToTop();
 		}
 	},
 
+	handleClickOutside: function() {
```

Comment:
It’s great that you extracted the code into a better named method. You also needed it in another place, but still :)